### DISCUSSION TITLE
Documentation question (a forum learning experience for me)

### DISCUSSION THREAD

**Discussion by ClaytonCurtis1947** [freeplane/freeplane#1226] [Help] [answered]
The opening page of Freeplane documentation ( https://docs.freeplane.org/) has a sentence that begins 
"Freeplane is also a more robust and superuser alternative to (...)"  Should "superuser" be changed to "superior" or left as is?

The reason I ask this is to help clarify for me what things are appropriate to bring to the community. It's pretty clear that I can make a change to an obvious typo or other similar class of error on my own fork and make a pull request, but when there's some degree of ambiguity as is the case here, should I raise it in a discussion as done here, or should I just go ahead with a change and let the developers decide?

Thanks.

**Comment by dpolivaev** [comment]
Do whatever improvements you believe to be right.
If it was only for me, I would just suggest that you commit your changes directly to the main branch that would deploy them.
But because @macmarrum wants to spend his energy on checking every piece of improvement we agreed that all changes are submitted as pull requests so that he has a chance of reviewing them.

**Comment by ClaytonCurtis1947** [comment]
Thanks very much. I hope I’m not causing more problems than my contributions will ever be worth!!

 

Clayton Curtis

**Comment by HaaiHenkie** [comment, marked as answer]
Hi Clayton,

I think what is meant here is "power user", a user that uses the more advanced features of a program as compared to an average user. Whether Freeplane is "superior" depends on who you ask. I think "superuser" is sometimes used as a synonym of "power user", but usually it has the meaning of a special user account used for system administration, like "root" or "admin". So I agree with you that it should be replaced.

Greetings,

Henk
